src/web/components/ParticipantManagement/EditParticipantDialog.stories.tsx

```diff
@@ -2,15 +2,15 @@ import type { Meta, StoryObj } from '@storybook/react';
 
 import { ParticipantStatus } from '../../../api/entities/Participant';
 import { UserRole } from '../../../api/entities/User';
-import UpdateParticipantDialog from './UpdateParticipantDialog';
+import EditParticipantDialog from './EditParticipantDialog';
 
-const meta: Meta<typeof UpdateParticipantDialog> = {
-  component: UpdateParticipantDialog,
+const meta: Meta<typeof EditParticipantDialog> = {
+  component: EditParticipantDialog,
   title: 'Manage Participants/Update Participant Dialog',
```

Comment:
Perhaps we should rename the story as well? 
```
title: 'Manage Participants/Edit Participant Dialog',
```

Reply:
Good catch!